### PR TITLE
[Keymap] Added personal keymap for DZ68RGB

### PR DESCRIPTION
**Pull request by MatthewRobo**
## Description

Added personal keymap for DZ68RGB
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/dztech/dz65rgb/keymaps/matthewrobo/keymap.c [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+	if (IS_LED_ON(host_keyboard_leds(), USB_LED_CAPS_LOCK)) {
+		rgb_matrix_set_color(8, 0xFF, 0xFF, 0xFF);
+	}
+
+	uint8_t this_led = host_keyboard_leds();
+
+	if (!g_suspend_state && rgb_matrix_config.enable) {
+		switch (biton32(layer_state)) {
+		case _NAV:
+			if (this_led & (1 << USB_LED_NUM_LOCK)) {
```
Comment (fauxpark) [inline keyboards/dztech/dz65rgb/keymaps/matthewrobo/keymap.c]:
```suggestion
			if (IS_LED_ON(this_led, USB_LED_NUM_LOCK)) {
```

File: keyboards/dztech/dz65rgb/keymaps/matthewrobo/keymap.c [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+
+ESC, 1,   2,   3,   4,   5,   6,   7,   8,   9,   0,   -,   =,  BCK, DEL
+TAB, Q,   W,   E,   R,   T,   Y,   U,   I,   O,   P,   [,   ],    \,   -
+CPS, A,   S,   D,   F,   G,   H,   J,   K,   L,   COL, QOT,  RETURN,   +
+SFT, Z,   X,   C,   V,   B,   N,   M,   COM, DOT, SLS, SHIFT,    UP,   0
+CTL, GUI, ALT,        SPACEBAR,              ALT, FN, CTL, LFT, DWN, RIT
+*/
+void rgb_matrix_indicators_user(void)
+{
+	if (IS_LED_ON(host_keyboard_leds(), USB_LED_CAPS_LOCK)) {
```
Comment (fauxpark) [inline keyboards/dztech/dz65rgb/keymaps/matthewrobo/keymap.c]:
```suggestion
	if (IS_HOST_LED_ON(USB_LED_CAPS_LOCK)) {
```

File: keyboards/dztech/dz65rgb/keymaps/matthewrobo/keymap.c [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+#define TG_NKRO MAGIC_TOGGLE_NKRO
+#define _V_V_V_ KC_TRNS
+#define LT_CAPS LT(_NAV, KC_CAPS)
+extern bool autoshift_enabled;
+#define MT_APP  RALT_T(KC_APP)
+#define LM_LALT LM(_FNM, MOD_LALT)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+	[_QWERTY] = LAYOUT_65_ansi(
+		KC_GESC, KC_1,     KC_2,     KC_3,   KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,     KC_MINS,  KC_EQL,  KC_BSPC, KC_DEL,  \
```
Comment (fauxpark) [inline keyboards/dztech/dz65rgb/keymaps/matthewrobo/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

Comment (fauxpark) [reply keyboards/dztech/dz65rgb/keymaps/matthewrobo/keymap.c]: I removed them from the templates ;)

**Comment by drashna** [comment]
Thanks!

**Pull request merged**